docker-compose.yml

```diff
@@ -72,14 +72,17 @@ services:
       - host.docker.internal:${DOCKER_HOST:-host-gateway}
     ports:
       - "3001:3001"
+    command: "sh -c /middleware.entrypoint.sh"
     volumes:
+      - uploads:/uploads
+      - ./dev/docker/middleware.entrypoint.sh:/middleware.entrypoint.sh
       - /var/tmp/ocis:/var/tmp/ocis
-      - ./tests/acceptance/filesForUpload:/usr/src/app/filesForUpload
     environment:
       MIDDLEWARE_PORT: 3001
       MIDDLEWARE_HOST: "0.0.0.0"
       RUN_ON_OCIS: 'false'
       BACKEND_HOST: "http://host.docker.internal:8080"
+      REMOTE_UPLOAD_DIR: "/uploads"
 
   middleware-ocis:
     image: owncloud/owncloud-test-middleware:latest
```

Comment:
The middleware version needs to be bumped as well

Reply:
should not be necessary as it is already using the `latest`